server/src/test/middleware/auth.middleware.test.ts

```
@@ -0,0 +1,110 @@
+    await verifyToken(req as Request, res as Response, next);
+
+    expect(res.status).toHaveBeenCalledWith(401);
+    expect(res.json).toHaveBeenCalledWith({ message: 'Unauthorized Access' });
+    expect(next).not.toHaveBeenCalled();
+  });
+
+  it('should return 500 if JWT_SECRET_KEY is missing', async () => {
+    req.cookies = req.cookies || {};
+    req.cookies.token = 'fake-token';
+    const originalEnv = process.env.JWT_SECRET_KEY;
+    delete process.env.JWT_SECRET_KEY;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)